app/scripts/controllers/deployment.js

```diff
@@ -260,6 +269,28 @@ angular.module('openshiftConsole')
           Logger.log("builds (subscribe)", $scope.builds);
         }));
 
+        function updateFilterWarning() {
+          if (!LabelFilter.getLabelSelector().isEmpty() && $.isArray($scope.replicaSetsForDeployment) && !$scope.replicaSetsForDeployment.length && !$.isEmptyObject($scope.unfilteredReplicaSetsForDeployment)) {
+            $scope.alerts["deployments"] = {
```

Comment:
I'm sure this is what we used on the other page, but I suggest a more descriptive ID like

```js
$scope['filter-hiding-all'] = ...
```